### ISSUE TITLE
SNMP Sensors 1.3.4 never starts on HASS Boot - has to be started manually

### ISSUE THREAD

**Issue by vk5ztv** [PecceG2/Home-Assistant-SNMP-Sensor-Server#6]
Running HASS 2024.6.4, HASS OS 12.4, SNMP Sensor Server add-on v1.3.4. Start At Boot, Watchdog and Auto Update are all ON. After restarting or rebooting HASS (which is a VM running under Proxmox 8.2.4), the add-on always fails to start and has to be started manually.

**Comment by PecceG2** [comment]
Hi @vk5ztv, thanks for reporting this bug.
I working now in this issue.

**Comment by PecceG2** [comment]
Debug error:
```
Executing busybox-1.36.1-r29.trigger
OK: 192 MiB in 82 packages
Traceback (most recent call last):
  File "//snmpd-configurator.py", line 21, in <module>
    ha_sensors = json.loads(ha_sensors_request.text)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.12/json/__init__.py", line 346, in loads
    return _default_decoder.decode(s)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.12/json/decoder.py", line 340, in decode
    raise JSONDecodeError("Extra data", s, end)
json.decoder.JSONDecodeError: Extra data: line 1 column 4 (char 3)
s6-rc: info: service legacy-services: stopping
s6-rc: info: service legacy-services successfully stopped
s6-rc: info: service legacy-cont-init: stopping
s6-rc: info: service legacy-cont-init successfully stopped
s6-rc: info: service fix-attrs: stopping
s6-rc: info: service fix-attrs successfully stopped
s6-rc: info: service s6rc-oneshot-runner: stopping
s6-rc: info: service s6rc-oneshot-runner successfully stopped
```

**Comment by PecceG2** [comment]
This issue is fixed in version **1.3.5**, please update to avoid this problem in the future.
